config/test.toml

```diff
@@ -1,9 +1,10 @@
 owner = "bnjbvr"
-dry_run = true
+dry_run = false
```

Comment:
Was this change intentional? I don't where this config is used but feel dangerous to change 😬

Reply:
Yeah this is my personal testing configuration on https://github.com/bnjbvr/octotest, a test repo dedicated for that, so it's fiiiiine :stuck_out_tongue: